scout/server/blueprints/variant/controllers.py

```diff
@@ -434,6 +434,8 @@ def observations(store, loqusdb, case_obj, variant_obj):
 
     if not obs_data:
         obs_data["total"] = loqusdb.case_count(variant_category=category, loqusdb_id=loqusdb_id)
+        if obs_data["total"]:
+            obs_data["observations"] = 0
```

Comment:
👍🏻

Reply:
Something of an afterthought, but it feels rather safe and in line with the earlier fix.